### ISSUE TITLE
Reuse the messages set into InputFilter validators

### ISSUE THREAD

**Issue by cbichis** [bramstroker/zf2-form#48]
Hi,
I think it would be interesting to not create new messages but rather reuse the messages from inputFilter.

I created such approach on my fork cbichis/zf2-form

**Comment by bramstroker** [comment]
Yes, good idea. I don't see a reason why we shouldn't utilize those. PR's are welcome

**Comment by cbichis** [comment]
PR created.

**Comment by cbichis** [comment]
Have you checked the PR ?

**Comment by bramstroker** [comment]
Yes I did already put my comments 21 days ago. Did you miss them?
See https://github.com/bramstroker/zf2-form/pull/50
